__tests__/endpoints.test.js

```diff
@@ -450,4 +450,49 @@ describe("GET /api/articles (topic query)", () => {
             expect(body.body).toEqual([])
         })
     })
+})
+
+describe("GET /api/articles/:article_id ADD COMMENT_COUNT", () => {
+    test("200: returns with article including comment_count", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({body}) => {
+            console.log(body)
+            const article = body
+            expect(article).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
```

Comment:
i'd probably add an expect for the exact comment count here too like you did for article 8 (using the seed data to count what it should be)